v2/tools/generator/internal/astmodel/resource_type.go

```diff
@@ -18,13 +18,16 @@ import (
 	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/astbuilder"
 )
 
+// ResourceKind is the scope the resource is deployed at.
 type ResourceKind string
```

Comment:
Should this then be named `ResourceScope` instead?

Reply:
I've changed this name (and the corresponding one in genruntime) to scope, as that more closely reflects what it actually represents.